x-pack/plugins/security_solution/scripts/endpoint/common/fleet_server/fleet_server_services.ts

```diff
@@ -0,0 +1,673 @@
+/*
```

Comment:
FYI: I will have a follow up PR to refactor the existing `endpint_agent_runner` utility to use this module (will also look for `import`'s for private modules of  `endpoint_agent_runner`). For now, most of the code here is a duplicate.